src/material-experimental/mdc-snack-bar/snack-bar-container.ts

```diff
@@ -100,20 +109,29 @@ export class MatSnackBarContainer extends BasePortalOutlet
    */
   @ViewChild('label', {static: true}) _label: ElementRef;
 
+  /**
+   * The div container that is hidden from screen readers and initially holds to snack bar
+   * content.
+   */
+  @ViewChild('nonLive', {static: true}) _nonLiveElementRef: ElementRef<HTMLElement>;
+
+  /** The div container that is a live region and will have the snack bar content move into. */
+  @ViewChild('live', {static: true}) _liveElementRef: ElementRef<HTMLElement>;
```

Comment:
We should avoid adding new code that depends on static queries. As far as I can tell these don't have to be static since they're accessed after `ngAfterViewInit`.

Reply:
Good point, you are correct that static isn't needed. Removed from both types of snack bar container.